### DISCUSSION TITLE
How should we generate 'Initial Sequence Number'

### DISCUSSION THREAD

**Discussion by Nayoung-Oh** [ANLAB-KAIST/KENSv3#64] [Project Q&A 2. Transport Layer (TCP)] [answered]
I checked that the grader does not check the initial sequence number, but I was just curious about whether should we follow some basic rules for generating initial sequence number like ([RFC 793](https://www.rfc-editor.org/rfc/rfc793#page-24)).
Or is there any given functions related to it or can we use a fixed number without considering the security issue?
Thanks in advance!

**Comment by Meraroo** [comment, marked as answer]
I'm no a TA, but I guess it can be any number.
May be you can utilize rand ().
But I'm not sure.....

**Comment by Pusnow** [comment]
You can use random number.
